src/main/java/uk/gov/hmcts/cmc/claimstore/services/pilotcourt/PilotCourt.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.cmc.claimstore.services.pilotcourt;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import uk.gov.hmcts.cmc.claimstore.services.ccd.legaladvisor.HearingCourt;
+
+import java.util.Optional;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class PilotCourt {
+    private String id;
+    private String postcode;
+    private Optional<HearingCourt> hearingCourt;
```

Comment:
Generally recommended not to use `Optional` as fields

Reply:
So use null as field and return Optional in getter?